extensions/wikia/PortableInfobox/tests/PortableInfoboxParserTagControllerTest.php

```diff
@@ -232,13 +247,13 @@ public function paramsDataProvider() {
 		return [
 			[ [ 0 => 'zero', 1 => 'one', 2 => 'two' ], [ 'zero', 'one', 'two' ] ],
 			[ [ 1 => 'three', 2 => 'four', 3 => 'five' ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "three" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)